app/controllers/api/v1/base_api_controller.rb

```diff
@@ -18,6 +19,7 @@ def authenticate_user_from_token!
         if user && Devise.secure_compare(user.authentication_token, params[:user_token])
             sign_in user, store: false
             renew_authentication_token(user)
+            puts(user.authentication_token)
```

Comment:
Line can be deleted

Reply:
Hm. This file/code is for the ThinQ.tv android app. Not sure if that line was for debugging purposes - I doubt the app needs it.